Library/Formula/kitchen-sync.rb

```diff
@@ -0,0 +1,23 @@
+class KitchenSync < Formula
+  desc "Fast efficiently sync database without dumping & reloading"
+  homepage "https://github.com/willbryant/kitchen_sync"
+  url "https://github.com/willbryant/kitchen_sync/archive/0.37.tar.gz"
+  sha256 "a78206cdd3ac4d4aa646cdfc8ae34c5df87179223c59d5dcca625760c68ae4e7"
+  head "https://github.com/willbryant/kitchen_sync.git"
+
+  depends_on "cmake" => :build
+  depends_on "boost"
+  depends_on :mysql => :recommended
+  depends_on :postgresql => :optional
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+
+  test do
+    assert File.exist?("#{bin}/ks")
+    assert File.exist?("#{bin}/ks_mysql") if build.with? "mysql"
+    assert File.exist?("#{bin}/ks_postgresql") if build.with? "postgresql"
```

Comment:
@bfontaine There is no argument for `-h (--help)`, we can connect to the database you want to synchronize or run the tests work with base.